src/display/editor/tools.js

```diff
@@ -740,32 +738,41 @@ class AnnotationEditorUIManager {
     }
   }
 
+  /**
+   * Add an editor the current selection.
+   * @param {AnnotationEditor} editor
+   */
+  addToSelection(editor) {
```

Comment:
This feels like a slightly misleading method name, since it either adds *or* removes a selection below. So, how about the following instead?
```suggestion
  /**
   * Add or remove an editor in the current selection.
   * @param {AnnotationEditor} editor
   */
  toggleSelected(editor) {
```